cxplat/inc/cxplat/common.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+#include <assert.h>
+#include <sal.h>
+
+#ifdef CXPLAT_SOURCE
+#define CXPLAT_API __declspec(dllexport)
+#else
+#define CXPLAT_API __declspec(dllimport)
+#endif
```

Comment:
Do you really want to support building cxplat as a DLL? If not, shouldn't all these be unnecessary?

Reply:
Leftover from usersim, I just did a global search/replace after moving files around.  No need to keep this, this is just WIP.

Reply:
Removed